js/DropDownTree/Checkbox.md

```diff
@@ -434,4 +434,4 @@ function onCheckUncheckAll(args) {
     
 {% endhighlight %}
 
-![](Checkbox_images/Checkbox_img2.png)
\ No newline at end of file
+![checkNode/uncheckNode in JavaScript DropdownTree](Checkbox_images/Checkbox_img2.png)
```

Comment:
CheckNode and uncheckNode in JavaScript DropdownTree

Reply:
Done